### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by UMiho**
## What kinds of PR

- [] New feature
- [] Improve feature
- [] Bug fix
- [x] Others

## What is this PR
added readme of obstacle stop planner. (The content is still based on obstacle stop planner before the refine.)

## How to check this PR
None.

### CODE REVIEW

**Comment by kenji-miyake** [comment]
Rebased and applied small fix.

**Comment by kenji-miyake** [comment]
@UMiho I updated the document(tried to keep as much as simple). Could you check this, please? :pray: 

[obstacle_stop_planner - Test Autoware Documentation.pdf](https://github.com/tier4/obstacle_stop_planner_refine/files/6197809/obstacle_stop_planner.-.Test.Autoware.Documentation.pdf)

**Comment by kenji-miyake** [comment]
@KeisukeShima Thank you! Could you check this error?
```
Error: Unable to resolve action `jnanadarshan/mkdocs-build-action@v1`, unable to find version `v1`
```

https://github.com/tier4/obstacle_stop_planner_refine/pull/6/checks?check_run_id=2191380948

File: .github/workflows/mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Set environment variable
+        id: plum
+        run: echo "::set-output name=PULL_NUMBER::$(jq --raw-output .pull_request.number $GITHUB_EVENT_PATH)"
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Build MkDocs
+        uses: jnanadarshan/mkdocs-build-action@v0.24
```
Comment (kenji-miyake) [inline .github/workflows/mkdocs.yml]: @KeisukeShima By the way why do you use this action instead of just using `pip`? :thinking:

Comment (KeisukeShima) [reply .github/workflows/mkdocs.yml]: @kenji-miyake I replaced script with pip :+1: Now it looks fine.

**Comment by KeisukeShima** [comment]
@kenji-miyake I fixed all errors. Now you can check documents here:
https://tier4.github.io/obstacle_stop_planner_refine/pr-check/pr6/site/packages/obstacle_stop_planner_refine/

File: .github/workflows/mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set environment variable
+        id: plum
+        run: echo "::set-output name=PULL_NUMBER::$(jq --raw-output .pull_request.number $GITHUB_EVENT_PATH)"
```
Comment (kenji-miyake) [inline .github/workflows/mkdocs.yml:15]: @KeisukeShima What is `plum`?

Comment (kenji-miyake) [reply .github/workflows/mkdocs.yml:15]: I heard from @KeisukeShima that this has no meaning. I'll create another PR to fix this.

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**